config/install/tide_webform.defaults.yml

```diff
@@ -1,4 +1,5 @@
 privacy_statement:
+  add_to_new_form: false
```

Comment:
Default to `FALSE`. Consumer sites need to enabled it themselves if they need this feature.